sandbox/src/EventMerge.js

```diff
@@ -1,28 +1,30 @@
 import React, { useRef, useEffect } from "react";
 
 export default function EventMerge() {
-  const eventMergeId = useRef(window.alloy("createEventMergeId"));
+  const eventMergeIdPromise = useRef(window.alloy("createEventMergeId"));
```

Comment:
Need to update the docs?

Reply:
Yeah, right here: https://github.com/AdobeDocs/alloy-docs/pull/50